pyrovider/services/provider.py

```diff
@@ -318,6 +318,21 @@ def _get_arg(self, ref: any):
 
         return ref # Literal
 
+    def _get_service(self, service_name: str):
+        try:
+            value = self.get(service_name)
+        
+        except Exception as e:
```

Comment:
Do we know what type of exception is expected here? If so, I prefer to be as specific as possible.